myning/objects/player.py

```diff
@@ -124,10 +124,6 @@ def add_available_exp(self, exp):
         else:
             self.add_experience(int(exp * self.macguffin.exp_boost))
 
-    def add_exp(self, exp):
```

Comment:
Why are we removing this? Aren't you calling it?

Reply:
Good catch, I meant to rename the `add_available_exp` to `add_exp`